src/subscriptions/walletConnect.ts

```diff
@@ -0,0 +1,177 @@
+import { WalletConnectProvider, BufferBlock, Block } from '../provider/types'
+
+export const WalletConnectSubscriptionsSymbol = Symbol('WalletConnectSubscriptions')
+
+export interface WalletConnectSubscriptions {
+    [WalletConnectSubscriptionsSymbol]: true
```

Comment:
Question about this syntax. Why is it `true` rather than `boolean`?

Reply:
More specific like that